app/experimenter/experiments/forms.py

```diff
@@ -851,28 +851,22 @@ class Meta:
 
     @property
     def required_reviews(self):
-        return (
-            self["review_science"],
-            self["review_advisory"],
-            self["review_engineering"],
-            self["review_qa_requested"],
-            self["review_intent_to_ship"],
-            self["review_bugzilla"],
-            self["review_qa"],
-            self["review_relman"],
-        )
+        required_review_names = self.instance.get_all_required_reviews()
```

Comment:
I think you could shorten this to just

    return [self[r] for r in self.instance.get_all_required_reviews()]